lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -1,6 +1,6 @@
 <div class="page-main row around-xs">
   <div class="col-xs" ng-controller="ImageSelectionController as image">
-    <div class="box text-center relative" os-dropzone="image.selectImage($file)">
+    <div class="box text-center relative" os-dropzone="image.selectImageByPath($file.path)">
```

Comment:
Amazing catch! Looks like I forgot to update this after refactoring `dropzone`. Fixed!